pkg/rawyaml/resources.go

```diff
@@ -21,7 +21,7 @@ func ToObjects(c *chart.Chart) (result []runtime.Object, _ error) {
 		}
 		objs, err := yaml.ToObjects(bytes.NewBuffer(resource.Data))
 		if err != nil {
-			return nil, err
+			continue
```

Comment:
Reading from https://github.com/rancher/fleet/pull/262/files#r570424566 to get some context, should we at least log the error before continuing?

Reply:
Any idea why I would not encounter this error running Rancher 2.5.2?

Reply:
@ron1 which version of fleet that you don't see this problem?